pkg/dao/schema/resource_state.go

```diff
@@ -0,0 +1,48 @@
+package schema
+
+import (
+	"entgo.io/ent"
+	"entgo.io/ent/schema/edge"
+	"entgo.io/ent/schema/field"
+
+	"github.com/seal-io/walrus/pkg/dao/entx"
+	"github.com/seal-io/walrus/pkg/dao/schema/mixin"
+	"github.com/seal-io/walrus/pkg/dao/types/object"
+)
+
+type ResourceState struct {
```

Comment:
Now we store state in pg, and plan in s3. What are the differences in size between the state and the plan for the same run? Why do we want to store them separately?

Reply:
Currently, s3 only store plan file, and the plan file will be removed as run fininshed. Shall we store state in s3 too?  We need parse state file to generate resource components.